HW18/HW18.cpp

```diff
@@ -0,0 +1,250 @@
+﻿#include <iostream>
+#include <string>
+
+const int TABLES_COUNT = 10;
+
+class Drink {
+public:
+    std::string name;
+    double price;
+
+    Drink(const std::string& name, double price) : name(name), price(price) {}
+    virtual ~Drink() = default;
```

Comment:
no prepare, order, cost virtual methods ?
Then there's no point in this hierarchy at all, just use one generic Drink all the time